static/app/views/performance/mobile/appStarts/screenSummary/index.tsx

```diff
@@ -109,120 +109,130 @@ function ScreenSummary() {
+                    allowZero: false,
+                    title:
+                      appStartType === COLD_START_TYPE
+                        ? t('Cold Start (%s)', PRIMARY_RELEASE_ALIAS)
+                        : t('Warm Start (%s)', PRIMARY_RELEASE_ALIAS),
+                    dataKey: `avg_if(span.duration,release,${primaryRelease})`,
+                  },
+                  {
+                    unit: DurationUnit.MILLISECOND,
+                    allowZero: false,
+                    title:
+                      appStartType === COLD_START_TYPE
+                        ? t('Cold Start (%s)', SECONDARY_RELEASE_ALIAS)
+                        : t('Warm Start (%s)', SECONDARY_RELEASE_ALIAS),
+                    dataKey: `avg_if(span.duration,release,${secondaryRelease})`,
+                  },
+                  {
+                    unit: 'percent_change',
+                    title: t('Change'),
+                    dataKey: `avg_compare(span.duration,release,${primaryRelease},${secondaryRelease})`,
+                  },
+                  {
+                    unit: 'count',
+                    title: t('Count'),
+                    dataKey: 'count()',
+                  },
+                ]}
+                referrer="api.starfish.mobile-startup-totals"
+              />
+            </HeaderContainer>
+            <ErrorBoundary mini>
+              <AppStartWidgets additionalFilters={[`transaction:${transactionName}`]} />
+            </ErrorBoundary>
+            <SamplesContainer>
+              <SamplesTables transactionName={transactionName} />
+            </SamplesContainer>
+            {spanGroup && spanOp && appStartType && (
+              <ScreenLoadSpanSamples
+                additionalFilters={{
+                  [SpanMetricsField.APP_START_TYPE]: appStartType,
+                  ...(deviceClass ? {[SpanMetricsField.DEVICE_CLASS]: deviceClass} : {}),
+                }}
+                groupId={spanGroup}
+                transactionName={transactionName}
+                spanDescription={spanDescription}
+                spanOp={spanOp}
+                onClose={() => {
+                  router.replace({
+                    pathname: router.location.pathname,
+                    query: omit(
+                      router.location.query,
+                      'spanGroup',
+                      'transactionMethod',
+                      'spanDescription',
+                      'spanOp'
+                    ),
+                  });
+                }}
+              />
+            )}
+          </Layout.Main>
+        </Layout.Body>
+      </PageAlertProvider>
+    </Layout.Page>
+  );
+}
+
+function PageWithProviders() {
+  const location = useLocation<Query>();
+
+  const {transaction} = location.query;
+
+  return (
+    <ModulePageProviders
+      title={[transaction, ROUTE_NAMES['app-startup']].join(' — ')}
+      baseURL="/performance/browser/app-startup"
```

Comment:
```suggestion
      baseURL="/performance/mobile/app-startup/spans"
```

I think this is the appropriate path for this file. This component renders the drill down into a screen which uses the `/performance/mobile/app-startup/spans` URL. Unless this component uses `baseURL` as the module list page then the URL should end with `/spans`